.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,28 @@
+const straightQuotes = require('./packages/markdownlint-rule-mui/straight-quotes');
+
+module.exports = {
```

Comment:
This is still compatible with https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint.